src/components/SignUpForm/styles.js

```diff
@@ -0,0 +1,14 @@
+const styles = () => ({
+  nameFields: {
+    marginBottom: '8px',
```

Comment:
in the scenarios that I can think of, we shouldn't ever ever have to hardcode any margin in terms of px

Reply:
These bottom margins are there so that all the Formik's <Field /> are spaced out evenly from each other. Any other way to do it?

Reply:
nvm fixed it, eliminated most hardcoded margin and padding